wiki/en/Running-a-Server.md

```diff
@@ -135,64 +115,65 @@ There is one upstream (musician sending to the server) and one downstream (serve
 
 ### Chat Welcome Message
 
-The text entered here appears to all users when they join the server (the chat window will open automatically for them).
+The text entered here appears to all users when they join the Server (the chat window will open automatically for them). HTML is also supported.
 
 ## Options
 
-
-<figure><img src="{% include img/en-screenshots/server-window-options.inc %}" style="width:80%; border:5px solid grey;" loading="lazy" alt="Image of Jamulus server window options"></figure>
+<figure><img src="{% include img/en-screenshots/server-window-options.inc %}" style="width:80%; border:5px solid grey;" loading="lazy" alt="Image of Jamulus Server window options"></figure>
 
 ### Recording Directory
 
-This sets the path to where server's recording will be stored. With this path set, the "Enable Jam Recoder" function in the Server Setup tab will make recording start once the first person connects to the server, and stops when the last person leaves. Use the "New Recording" button to create a new sub-directory in which the recordings will be stored from then on. Note that Recordings are per track in [Audacity](https://www.audacityteam.org/) `.lof` format and [REAPER](https://en.wikipedia.org/wiki/REAPER) `.rpp`. Open the respective files to listen to them in those applications.
+This sets the path to where the Server's recordings will be stored. With this path set, the "Enable Jam Recoder" function in the Server Setup tab will make recording start once the first person connects to the Server, and stops when the last person leaves. Use the "New Recording" button to create a new sub-directory in which the recordings will be stored from then on. Note that Recordings are per track in [Audacity](https://www.audacityteam.org/) `.lof` format and [REAPER](https://en.wikipedia.org/wiki/REAPER) `.rpp`. Open the respective files to listen to them in those applications.
 
-**Note**: When your server is recording, clients will display a message that recording is on.
+**Note**: When your Server is recording, clients will display a message that recording is on.
 
-### Delay panning
+### 
+
+Directory Server Address
 
-This option uses small differences in sound arrival time between the two ears. It produces a stereo effect similar to natural human hearing when compared to normal "volume" panning. Note that to achieve the effect, all clients should be set to 'mono in / stereo out' (or to 'stereo', but only if transmitting a stereo signal).
+Leave this field empty unless you need to list your Server on a [Custom Directory](#3-directory) or run a Directory.
 
-### Custom Directory Server Address
+### Server List Filename
 
-Leave this field empty unless you need to list your server on a [private Directory](#3-directory)
+Leave this field empty unless you need to run your Server as a [Directory](#3-directory)
```

Comment:
Add:
> When in use, this will hold the list of registered Servers whilst restarting the Directory.  This prevents the server list appearing "empty" until the Servers reregister.